### ISSUE TITLE
Error with remote theme cloning with git bash on windows

### ISSUE THREAD

**Issue by damsfx** [OFFLINE-GmbH/oc-bootstrapper#22]
Hi, 

Everything work well when command is ran from vagrant's environment (ssh connected to box).  
If I run the `october install` in a git bash environment I get this error :

````
Installing Theme...
Exit code: 1 while executing: "LC_ALL=C git clone "git@gitlab.com:acme/oc-acme-template.git" "C:\Users\...\mysite.local\themes\acme"" with reason: 'LC_ALL'  is not recognized as an internal or external command
````

**Comment by tobias-kuendig** [comment]
I had to wait on [this](https://github.com/matteosister/GitElephant/pull/143) to be merged an released. Looks like a new verion of GitElephant is available. I'll release a new bootstrapper version with this fix.

**Comment by tobias-kuendig** [comment]
This should actually already be released in 0.4.2 of bootstrapper.

Can you do a `composer global update` and check `october -v` is `0.4.2`?

Then try again.

**Comment by damsfx** [comment]
Hi @tobias-kuendig 

My version of october is up to date : v0.4.2.
